### PR TITLE
Migrate to Emotion

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: patches/babel-plugin-emotion+10.2.2.patch
Comment (kylegach) [inline patches/babel-plugin-emotion+10.2.2.patch:1]: Can you share more about this patch?

Comment (Andarist) [reply patches/babel-plugin-emotion+10.2.2.patch:1]: This basically makes the plugin to handle styled imports from your theming package - instead of the `@emotion/styled` pkg.

Most importantly this makes "components as selectors" to work. In v11 we have a babel plugin option to tell us which imports are just wrapper/proxies to our APIs so that could be just configured in the v11. But since you are stuck at v10 i had to hack this with patch-package in all of your repos

Comment (kylegach) [reply patches/babel-plugin-emotion+10.2.2.patch:1]: Aha. Thanks for explaining! 👍

**Review by kylegach — APPROVED** [review]
Thanks, @Andarist! Changes look good to me. 👍

**Comment by winkerVSbecks** [comment]
@kylegach is this good to merge now or should we hold off until the versioning docs branch is merged

**Comment by kylegach** [comment]
I'd like to merge #310 first, because that includes a workflow on push to `master`, which we can test with this one.

I'll handle the necessary updates to this PR.

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
@winkerVSbecks WDYT, can we merge this?

**Comment by winkerVSbecks** [comment]
@ndelangen yea, let me fix the linting errors and then I'll merge it

**Pull request merged**
